static/include/how-to/gather-sync-sensor.md

```diff
@@ -0,0 +1,45 @@
+Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}} and see trends over time.
+
+{{< table >}}
+{{< tablestep link="/services/data/">}}
+{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
+**1. Add the data management service**
+
+On your machine's **CONFIGURE** tab, add the **data management** service.
```

Comment:
maybe more steps here like with the sensors?

Reply:
done